src/core/components/dataList/dataListFilter/dataListFilterStatus.tsx

```diff
@@ -20,28 +23,35 @@ export const DataListFilterStatus: React.FC<IDataListFilterStatusProps> = () =>
     }
 
     return (
-        <p
-            className={classNames(
-                'px-3 text-sm font-normal leading-tight text-neutral-500 md:text-base',
-                { 'text-primary-400': isLoading || isInitialLoading },
-                { 'text-neutral-500': displayItemsCount },
-            )}
-        >
-            {displayItemsCount && (
-                <>
-                    <span className="text-neutral-800">{itemsCount} </span>
-                    <span>{entityLabel}</span>
-                </>
-            )}
-            {isInitialLoading && copy.dataListFilterStatus.loadingEntity(entityLabel)}
-            {isLoading && copy.dataListFilterStatus.filteringEntity(entityLabel)}
+        <div className="flex h-10 items-center justify-between">
+            <p
+                className={classNames(
+                    'px-3 text-sm font-normal leading-tight text-neutral-500 md:text-base',
+                    { 'text-primary-400': isLoading || isInitialLoading },
+                    { 'text-neutral-500': displayItemsCount },
+                )}
+            >
+                {displayItemsCount && (
+                    <>
+                        <span className="text-neutral-800">{itemsCount} </span>
+                        <span>{entityLabel}</span>
+                    </>
+                )}
+                {isInitialLoading && copy.dataListFilterStatus.loadingEntity(entityLabel)}
+                {isLoading && copy.dataListFilterStatus.filteringEntity(entityLabel)}
+                {isFiltered && (
+                    <>
+                        <span>{copy.dataListFilterStatus.found} </span>
+                        <span className="text-primary-400">{itemsCount} </span>
+                        <span>{entityLabel}</span>
+                    </>
+                )}
+            </p>
             {isFiltered && (
```

Comment:
We should also check that the `onResetFiltersClick` property is defined. I would also add a test to make sure that the button is not rendered when `onResetFiltersClick` is not defined

Reply:
Good catch - thanks